security/unittest/rolestest.cpp

```diff
@@ -67,16 +68,9 @@ class TestRolesWithServer : public testing::Test {
 #endif /* OC_DYNAMIC_ALLOCATION */
   }
 
-  static void TearDownTestCase()
-  {
-    oc::TestDevice::StopServer();
-  }
-
   void TearDown() override
   {
-    for (auto &peer : peers_) {
-      oc_tls_remove_peer(&peer->endpoint);
```

Comment:
`oc_tls_remove_peer` is no longer sufficient to delete the peer and remove associated roles certs? Reseting the server costs us 200ms so I usually want to do it just once at the start if possible. (This has 10 test cases -> now takes ~2 secs longer)

Reply:
Issue is that the TLS `oc_tls_remove_peer` is called in RFOTM state so  reset has been executed, and device after reset drops commands for  for 2 seconds

Reply:
reverted